### ISSUE TITLE
Instant Crashes on Open

### ISSUE THREAD

**Issue by AhoyCapn** [Chatterino/chatterino2#1571]
**Describe the bug**
When I open Chatterino, it instantly crashes. 

**Chatterino version**
Chatterino 2.1.7 (Not sure the specific version because of the crashing. I downloaded the installer through the download button on the Chatterino website. Will update when my Chatterino is working again.)

**Operating system**
Windows 10 Pro Version 1903 (OS Build 18362.356)

**Additional information**
The instant crashing will sometimes fix itself. For me, it does not matter if I reinstall the program, restart my computer, run as administrator, or delete the %appdata% files. The bug will occur for me most often when I start up my computer, however, it has happened for me before where I close Chatterino and reopen it.

**Screenshot**
The screenshot attached is what Chatterino opens up to before crashing right after. 

![Screenshot_197](https://user-images.githubusercontent.com/47093949/75305014-1abd1680-5802-11ea-8a65-92fe0a1ee3dd.png)

**Comment by alazymeme** [comment]
This sounds like an incredibly random bug, which may be due to Windows being literal garbage. Do you have any steps to try and reproduce? I'm currently on the latest nightly on windows 10 pro with zero issues. Do you have the x86/x64 redists installed? https://support.microsoft.com/en-au/help/2977003/the-latest-supported-visual-c-downloads

**Comment by AhoyCapn** [comment]
> This sounds like an incredibly random bug, which may be due to Windows being literal garbage. Do you have any steps to try and reproduce? I'm currently on the latest nightly on windows 10 pro with zero issues. Do you have the x86/x64 redists installed? x86: https://aka.ms/vs/16/release/vc_redist.x86.exe x64: https://aka.ms/vs/16/release/vc_redist.x64.exe (my general habit is to install both, despite only being on a 64bit pc)

I have them both installed yea. I cannot reproduce it in anyway, as it happens every once awhile and randomly. My only solution to fixing it was to delete all of the chatterino files in its entirety, physically shutdown my computer, and run an attempt to an update that does not even work. I wouldn't be surprised if it was just because how shitty windows can be.

**Comment by pajlada** [comment]
@AhoyCapn Did you fix this by deleting your settings folder?

**Comment by AhoyCapn** [comment]
> @AhoyCapn Did you fix this by deleting your settings folder?

I explained in the original post that deleting the settings folder did not fix the issue.

**Comment by packetsinc** [comment]
I have the same problem on chatterino 2.1.7 under Win 10 Pro 1903 build 18362.657, except it happens every time I launch it regardless of recent reboots. I have also tried fresh installs, deleting the settings folder, and launching as admin with no change in behavior. I didn't change any settings or add any channels for more than a week before first occurrence.

This has been happening for about a week or two, and my last windows update was installed about a month ago.

**Comment by th3an7** [comment]
Also getting crashes but for me it happens randomly... sometimes after 5 minutes after launch and sometimes I can run it everyday for whole week without crashes - dunno if it the same issue...
Doesn't matter which version I use - although I switch every month or so using automatic builds - still happens on almost every single one I've been using so far

I've also noticed that it logs `Exception code: 0xc0000094` in event viewer
example log in event viewer:
```
Faulting application name: chatterino.exe, version: 0.0.0.0, time stamp: 0x5e6ccb35
Faulting module name: chatterino.exe, version: 0.0.0.0, time stamp: 0x5e6ccb35
Exception code: 0xc0000094
Fault offset: 0x000000000009bd3d
Faulting process id: 0x11b7c
Faulting application start time: 0x01d5fd86a1218b56
Faulting application path: C:\Programy\Chatterino2-dev\chatterino.exe
Faulting module path: C:\Programy\Chatterino2-dev\chatterino.exe
Report Id: 23f32529-28ff-43d2-b633-c1d0759b7e61
Faulting package full name: 
Faulting package-relative application ID:
```
Can open new issue if needed

**Comment by packetsinc** [comment]
This seems to have self-resolved for me with windows build 18362.720

**Comment by fourtf** [comment]
not reproducible
